test/oas/from-open-api.test.ts

```diff
@@ -0,0 +1,71 @@
+// @vitest-environment happy-dom
+import {
+  fromOpenApi,
+  MapOperationFunction,
+} from '../../src/open-api/from-open-api.js'
+import { createOpenApiSpec } from '../support/create-open-api-spec.js'
+import { InspectedHandler, inspectHandlers } from '../support/inspect.js'
+
+it('creates handlers based on provided filter', async () => {
```

Comment:
Let's add another test case for _mapping_ the operation. It may look like this:

1. OAS with 1 operation defined.
2. In the map function, we modify something about the operation (e.g. its path).
3. The rest of the test as usual (request -> assertion).

Reply:
That's a great test! 45df8944b7508871660d9e84dfe7381eed3fbca8
I modified response, since operation itself does not contain `path` and that one we are not able to modify in this mapping function